SPECS/mariner-repos/mariner-repos.spec

```
@@ -59,6 +69,14 @@ Requires:       %{name}-shared = %{version}-%{release}
 %description extended-preview
 %{summary}
 
+%package extended-debug-preview
+Summary:        CBL-Mariner Extended Debuginfo preview repo file.
+Group:          System Environment/Base
+Requires:       %{name}-shared = %{version}-%{release}
+
+%description extended-debug-preview
+%{summary}
+
```

For package names, since it looks like we already started with `mariner-repos-debug-[repo_name]`, then I'd stick with it. Similarly, for the .repo file names, although that's less like to be visible for the users and probably won't change their experience at all.

```suggestion
%package debug-extended-preview
Summary:        CBL-Mariner Extended Debuginfo preview repo file.
Group:          System Environment/Base
Requires:       %{name}-shared = %{version}-%{release}

%description debug-extended-preview
%{summary}

```

---

not going to make any changes here.  I actually think it should go the other way, but we'll fix this in Mariner 3.0.